src/components/modal/createModal/CreateModal.jsx

```diff
@@ -35,19 +37,32 @@ class CreateModal extends React.Component {
 
   create = async () => {
     this.setState({ showError: true });
-    return this.props.create(this.state.createParametersState);
+    if (this.state.createPassword) {
+      return this.props.create({ ...this.state.createParametersState, password: undefined }, this.state.createPassword);
```

Comment:
Don't pass it if it is undefined?

Reply:
The password is in the createParameterState, so I overwrite it to be empty